calcom/cal.com
# fix: managed event type string

## What does this PR do?

Fixes #12883 
Fix the text "count_members" from the team's managed event types. We don't have "count_members" or "count_members_one" or "count_members_other" in the locales to set count.

[Demo](https://share.vidyard.com/watch/3VoZgXf547Er8M5srEQ9ye?)

## Type of change


## How should this be tested?



- Go to event types page
- Select managed event in the team event list or create new team event and select managed event
- Check "Assignment" tab

## Mandatory Tasks


## Checklist

- I haven't added tests that prove my fix is effective or that my feature works
